ioSPI/fourier.py

```diff
@@ -0,0 +1,212 @@
+import numpy as np
+import pyfftw.interfaces.numpy_fft
+from numba import jit
+
+
+@jit
+def neg_pos_2d(arr2d):
+    """Apply checkerboard pattern.
+
+    Each pixel switches from positive to negative in
+    checker board pattern.
+
+    Parameters
+    ----------
+    arr2d : numpy.ndarray, shape (N,N)
```

Comment:
This is inconsistent with line 23: `assert arr2d.ndim == 3`. Also, please define `N`.

Reply:
Good catch. It's actually three D because it works for batch. I'll put in `(n_particle,N_fourier_pixels,N_fourier_pixels)` so that people know what `N` is.

Reply:
Tests: https://github.com/compSPI/ioSPI/pull/17/commits/a06ee824814195e106e289be5ce8d36688c5e64b

See my question about the "test data" at the beginning. The functions all use this data.